.env.example

```diff
@@ -16,4 +16,4 @@ SITE_URL=***
 GOOGLE_RECAPTCHA_SITE_KEY=***
 
 # New server-side recaptcha key
```

Comment:
According to the comments - the other key is meant to be removed.
This PR doesn't touch the site_key, so I guess the comments are incorrect.... bes tto update them now